docs/Maintainers.rst

```diff
@@ -83,6 +83,10 @@ The EasyBuild maintainers all have admin access to the different EasyBuild GitHu
 
   * GitHub: `@lexming <https://github.com/lexming>`_ - e-mail: ``alex.domingo.toro (at) vub.be`` - IRC/Slack: ``lexming``
 
+* **Bob Dröge** (`University of Groning <https://www.rug.nl/?lang=en>`_)
```

Comment:
```suggestion
* **Bob Dröge** (`University of Groningen <https://www.rug.nl/?lang=en>`_)
```

Reply:
As somebody born in Grunnen I felt I could commit this suggestion :)

Reply:
That was a test. You passed. :-P